### ISSUE TITLE
Ensure "Enter" key press functions the same as a "continue" button click on all steps of the setup wizard

### ISSUE THREAD

**Issue by marcellamaki** [learningequality/kolibri#11458]
There seems to be some inconsistency in the "Enter" key functionality of the setup wizard, where in some places, using the "Enter" key works to advance the form no matter what, and in other places, it requires using keyboard navigation and tabbing to the "continue" (or other primary button), and enter only works as expected when the element is in focus. 

This should be solved by ensuring that when in the setup wizard, the enter key functions the same as a click on the primary button in all forms when not using keyboard navigation. It should work with keyboard navigation (using tab and seeing a focus outline appear) as well. When using keyboard navigation, the "enter" key should work as expected within the form element, and should not interfere with [expected keyboard navigation options](https://webaim.org/techniques/keyboard/).

All form field validations should be triggered on the enter key, just as with a "click" on the relevant primary button.

(cc @radinamatic for any further A11y notes or guidance to add)

**Comment by Nyu10** [comment]
Hi, I would like to tackle this issue and expect to put up a PR in the next 2-3 weeks

**Comment by marcellamaki** [comment]
Hi @Nyu10 - thanks for your interest! This issue has already been assigned to a member of our team. You can find additional available issues by looking for [issues with no assignee with the "help wanted" tag on Kolibri](https://github.com/learningequality/kolibri/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+no%3Aassignee) or on [Kolibri Studio](https://github.com/learningequality/studio/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+no%3Aassignee)

**Comment by Nyu10** [comment]
Sounds good, will look for another issue

**Comment by marcellamaki** [comment]
Fixed in https://github.com/learningequality/kolibri/pull/11537
